### PR TITLE
Adding ListEntries and DeleteLogger commands

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

**Comment by tmatsuo** [comment]
@bshaffer Please take a look.

@SurferJeffAtGoogle FYI

File: logging/src/BaseCommand.php [hunk 1]
```diff
@@ -39,6 +39,12 @@ protected function addProjectOption()
             'If omitted then the current gcloud project is assumed. ',
             $this->getProjectIdFromGcloud()
         );
+        $this->addOption(
+            'logger',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'name of the logger', 'my_logger'
```
Comment (bshaffer) [inline logging/src/BaseCommand.php]: nit: can we get `my_logger` on its own line for consistency?

Comment (bshaffer) [inline logging/src/BaseCommand.php]: Would be great to include a blurb as to why it's useful to name your logger.

File: logging/src/DeleteLogger.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class DeleteLogger
+ * @package Google\Cloud\Samples\Logging
+ *
+ * This command deletes a logger and all its entries.
+ */
+class DeleteLogger extends BaseCommand
```
Comment (bshaffer) [inline logging/src/DeleteLogger.php]: This should be suffixed with `Command` for consistency, e.g. `DeleteLoggerCommand`

File: logging/src/ListEntries.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class ListEntries
+ * @package Google\Cloud\Samples\Logging
+ *
+ * This command simply lists log messages.
+ */
+class ListEntries extends BaseCommand
```
Comment (bshaffer) [inline logging/src/ListEntries.php]: Same here - should be `ListEntriesCommand`

File: logging/src/Write.php [hunk 4]
```diff
@@ -30,7 +30,7 @@
  *
  * This command simply writes a log message via Logging API.
  */
-class Write extends CommandWithProject
+class Write extends BaseCommand
```
Comment (bshaffer) [inline logging/src/Write.php]: `WriteCommand`

File: logging/test/ListEntriesTest.php [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    /* @var $hasCredentials boolean */
+    protected static $hasCredentials;
+    /* @var $application \Symfony\Component\Console\Application */
+    private $application;
+    /* @var $projectId mixed|string */
+    private $projectId;
+    /* @var $loggerName string */
+    private $loggerName;
+    /* @var $message string */
+    private $message;
```
Comment (bshaffer) [inline logging/test/ListEntriesTest.php]: These variables are all overkill except for `$projectId` and `$hasCredentials`. We should just create the application each time and hardcode loggerName and message as strings.

File: logging/src/BaseCommand.php (hunk 1 above)
Comment (tmatsuo) [inline logging/src/BaseCommand.php]: Done (in the next push)

File: logging/src/DeleteLogger.php (hunk 2 above)
Comment (tmatsuo) [inline logging/src/DeleteLogger.php]: done

File: logging/src/ListEntries.php (hunk 3 above)
Comment (tmatsuo) [inline logging/src/ListEntries.php]: done

File: logging/src/Write.php (hunk 4 above)
Comment (tmatsuo) [inline logging/src/Write.php]: done

File: logging/test/ListEntriesTest.php (hunk 5 above)
Comment (tmatsuo) [inline logging/test/ListEntriesTest.php]: done

**Comment by tmatsuo** [comment]
@bshaffer Thanks PTAL

**Comment by bshaffer** [comment]
LGTM!

**Pull request merged**